packages/larva-patterns/objects/o-author/o-author.prototype.js

```diff
@@ -0,0 +1,13 @@
+const clonedeep = require( 'lodash.clonedeep' );
+
+const c_span = clonedeep( require( '../../components/c-span/c-span.prototype' ) );
+const c_timestamp = clonedeep( require( '../../components/c-timestamp/c-timestamp.prototype.js' ) );
+
+c_span.c_span_text = 'Author Name';
+
+module.exports = {
+	o_author_classes: 'lrv-u-font-size-14',
```

Comment:
Shouldn't this use one of the `lrv-u-font*` classes instead? The `lrv-u-font*` utility classes were deprecated in favor of the algorithm versions

Reply:
@ethitter I think you meant to write the `lrv-a-font`? I'll update.